react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default class CustomButton extends React.Component {
+  constructor(prop) {
+    super(prop);
+    this.state = {
+      className: 'none',
```

Comment:
since the class is derived from the counter, you can remove it from state and calculate it in `render` instead.